elasticsearch/includes/classes/class-dashboard.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Automattic\VIP\Elasticsearch;
+
+class Dashboard {
+
+	function __construct() {
+		add_action( 'admin_init', array( __CLASS__, 'admin_init' ) );
```

Comment:
As discussed in the comments, lets be sure that it's left enabled for Automatticians at this point.

Reply:
Yep, the dashboard still works. We're just hiding the navigation.

Reply:
I mean we should probably just leave the navigation - I've been using it pretty much constantly while setting up sites + testing.

Reply:
Ok, we can probably just wait to deploy this until we have something to replace it with then.

Reply:
@joshbetz what do you think about 02d31907?

Reply:
Sweet, that works. I didn't realize it would be so easy 🙃